### DISCUSSION TITLE
NetPad as a web app

### DISCUSSION THREAD

**Discussion by EmilySamantha80** [tareqimbasher/NetPad#277] [Q&A]
I read in the documentation that NetPad can be hosted as a web app (or am I way off on that?). Is it possible, and if so how? I couldn't find any clues.

**Comment by tareqimbasher** [comment]
You can find how to do that on the [CONTRIBUTING.md](https://github.com/tareqimbasher/NetPad/blob/main/CONTRIBUTING.md#web-browser) page. Let me know if you face any issues!
